test/sequential/test-util-debug.js

```diff
@@ -26,7 +26,7 @@ function test(environ, shouldWrite) {
 
   var spawn = require('child_process').spawn;
   var child = spawn(process.execPath, [__filename, 'child'], {
-    env: { NODE_DEBUG: environ }
+    env: { NODE_DEBUG: environ, HOME: process.env.HOME }
```

Comment:
It saves traces for userspace tracing into HOME by default, which seems to be taken from the env variable. If you spawn a child from within Node, the child doesn't have access to that env variable, and It breaks.